include/libtorrent/aux_/session_impl.hpp

```diff
@@ -1370,6 +1370,17 @@ namespace aux {
 			bool should_log() const override;
 			void debug_log(const char* fmt, ...) const noexcept override TORRENT_FORMAT(2,3);
 			session_interface& m_ses;
+
+#if TORRENT_USE_RTC
+			void generate_rtc_offers(int /*count*/
+				, std::function<void(error_code const&, std::vector<aux::rtc_offer>)> handler)
+			{
+				handler(boost::asio::error::operation_not_supported, {});
+			}
+			override void on_rtc_offer(aux::rtc_offer const&) {}
+			override void on_rtc_answer(aux::rtc_answer const&) {}
```

Comment:
`override` goes at the end of the function signature. The fact that CI is still green suggests that this code isn't exercised there. at least on GCC/Linux, this causes a compilation error:
```In file included from test_listen_socket.cpp:36:
../include/libtorrent/aux_/session_impl.hpp:1380:4: error: ‘override’ does not name a type
 1380 |    override void on_rtc_offer(aux::rtc_offer const&) {}
      |    ^~~~~~~~
```

Reply:
Weird, since I added a travis build with `variant=test_debug tests=1 crypto=openssl webtorrent=on toolset=gcc sanitizer=on`.